android/build.gradle

```diff
@@ -21,4 +25,4 @@ android {
 
 dependencies {
   compileOnly 'com.facebook.react:react-native:+'
```

Comment:
First of all, thanks for the contribution. I'm waiting for this PR to be merged. And It would be better if you replace "compileOnly" with "implementation".

Reply:
Hi @abdurrahmanekr, could you include a link that explains why that’s better?

Reply:
Hi @paulmelnikow, I was actually going to say what @gtebbutt said.

Reply:
To be merged this needs some documentation in the readme. Would you be up for adding that? You could make a new branch off master with these commits and then add the documentation.

Would be great to get this merged!

Reply:
Yes, I would like to contribute but I don't think the use of this should be written on README because it is now available on most plugins. Because the user does not need to make any settings. The "buildscript.ext" variable in the <project dir> /android/build.gradle file is actually "rootProject.ext". [This link](https://github.com/williambout/react-native-proximity/pull/13) can be an example. You want a sentence like I wrote?